package.json

```diff
@@ -75,7 +75,7 @@
     "broccoli-uglify-sourcemap": "^1.0.1",
     "co": "^4.6.0",
     "common-tags": "^1.4.0",
-    "ember-cli": "^3.1.4",
+    "ember-cli": "github:ember-cli/ember-cli#f724919b2d0455899411908531c9179240f5ef41",
```

Comment:
To make the tests pass, I had to use this hash. This is the `ember-cli` version used in `ember.js` until now: https://github.com/emberjs/ember.js/blob/master/package.json#L112. Here is the reason why we needed this specific version in `ember.js`: https://github.com/emberjs/ember.js/pull/16525.

What I suggest is keeping the PR this way while the work on https://github.com/ember-cli/ember-cli-blueprint-test-helpers/pull/153 is in progress. Thanks to the updated blueprints test helpers, we'll be able to use a tagged release of `ember-cli` (in both `ember.js` and `ember-data`) and improve the tests for the blueprints (in both repo).

Wdyt @runspired?

Reply:
Disclaimer: `github:ember-cli/ember-cli#f724919b2d0455899411908531c9179240f5ef41` is roughly `3.1.2`.

Reply:
cc @rwjblue this is the `ember-cli` version issue I ran into and that we discussed a few days ago.